R/db.R

```diff
@@ -91,3 +91,22 @@ setMethod("sqlCreateTable", "DB2/AIX64",
     ))
   }
 )
+
+# Microsoft SQL Server ---------------------------------------------------------
+
+# Simple class prototype to avoid messages about unknown classes from setMethod
+setClass("Microsoft SQL Server", where = class_cache)
```

Comment:
And this class gets correctly overridden if we're actually using the SQL server driver?

Reply:
Good question -
For any back end ( and any driver ) these catalog functions need their arguments to be stripped of the quoted delimiters ( according to the ODBC spec ), so I think this is a good fix.

If we were to somehow get this wrong, however, when writing to tables only `FreeTDS` ( or other drivers that suffer from similar deficiencies ) would be affected, since other drivers can recover gracefully from us no being able to describe the column properties of the target table.